### ISSUE TITLE
Add option to switch to last active, i.e. most recent, tab

### ISSUE THREAD

**Issue by vertigo220** [microsoft/terminal#13578]
# Description of the new feature/enhancement

As the title says. Currently can go to previous, next, and last tab, as well as directly to any one of the first several, but can't alternate between two. This is the standard behavior of Ctrl+Tab in many other applications (browsers, Notepad++, etc) and it would be nice to have the same ability here.

# Proposed technical implementation details (optional)

**Comment by zadjii-msft** [comment]
Perhaps you're looking for `"tabSwitcherMode": "mru"` - [on the action](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#open-next-tab), or even just [set globally](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/interaction#tab-switcher-interface-style)/?

**Comment by vertigo220** [comment]
Yes, thank you, changing that to mru was indeed the solution I was looking for. I suppose this is a case of just not looking deep enough for the solution but, to be fair, there was nothing in the settings either to help point me in the right direction or to even indicate that such a thing was possible. I realize this is a program meant for advanced users, but I still feel guidance is always helpful both to make things easier for the user and to help prevent issues like these.

I'd suggest starting with adding a prominent message at the top of the actions page (and perhaps all pertinent pages) notifying the user that the actions can be further modified by editing the JSON file and providing a link to https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions (or the appropriate URL for each settings page). This may very well have been enough to tell me the option I'm looking for doesn't exist in settings, but maybe I can manually set it that way.

To take it a step further and help even more, each action could have a link to the specific part of that URL pertinent to that action, e.g. "Next tab" and "Previous tab" could have a link to https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#open-next-tab and https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#open-previous-tab, respectively, and/or show the various options ("mru", "inOrder", "disabled" in this case) in a tooltip or on the right side in the empty space there, and maybe even in a dropbox in the edit mode to automatically change it in the JSON from there. This all would admittedly be a fair amount of work and not really necessary, as even just pointing users in the right direction will be a big help. For example, now that I know this, I know where to look to see if I can do something and how to do it if it's an available option, which will make a big difference, but there may still be times I, or others, might just not see something that looks like what I want or it might be helpful to be able to see at a glance what the options actually are. That all said, this may not even apply to most of the settings, and may not be very useful at all for all but a few of them, and is just me thinking out loud based on being new to Terminal, whereas those who are more familiar with it may very well recognize that fact as may I once I use it more, but I just wanted to offer my thoughts in case they're helpful.

**Comment by zadjii-msft** [comment]
Those are some good suggestions. Overall, our goal with the Actions page is to enable all the properties of an action to be customizable there. It's just a little bit more engineering time than we have the resources for at the moment. I think we're tracking that broadly in some combination of #12388, #10000, #6900. 

FWIW, the global setting is available on the Interaction page of the SUI: 

![image](https://user-images.githubusercontent.com/18356694/180799778-27ee6434-9161-4406-be33-66c442c2a64f.png)

I think I'm gonna tentatively call this a /dupe of #6900. We'd probably accept a smaller PR for just adding a link to the docs,. Ultimately, the goal is to just redo most of that page.
